### DISCUSSION TITLE
Restore old toc.intigrate behavior

### DISCUSSION THREAD

**Discussion by Subnet-Masked** [squidfunk/mkdocs-material#6226] [Q&A] [answered]
Is there a way to make the left side navigation not slide the current page to the right? The behavior causes page titles to wrap down if they are long... It appears that this behavior changed recently, it used to not do this until a recent update.

![image](https://github.com/squidfunk/mkdocs-material/assets/55600371/c675eb0d-5725-4c14-bb98-3a7050b1508d)
![image](https://github.com/squidfunk/mkdocs-material/assets/55600371/6e9be30c-253f-4ccb-a8f1-f43a3f72e8dd)

This image from the documentation site shows how it used to behave:
![image](https://github.com/squidfunk/mkdocs-material/assets/55600371/c700735f-eafe-4be5-b5e1-1301d031b82b)

**Comment by squidfunk** [comment]
Nothing changed. It's just when you use page icons, and the titles are aligned with them. If you don't like this style, you can add some additional CSS, to style the icon to be inline with the text.

**Comment by Subnet-Masked** [comment]
@squidfunk That was quick!

I just tried removing the icons from the pages to see if the behavior would change, but it did not... Maybe these is something I am not understanding or doing somewhere else causing it?

![image](https://github.com/squidfunk/mkdocs-material/assets/55600371/28c01a51-97e1-41d1-acba-a763bcd02f73)

**Reply by Subnet-Masked** [reply]
Looks like the behavior changed in 9.4.2 as when using 9.4.1 the text does not move when the page is active.

**Reply by squidfunk** [reply]
Okay then, I'm not sure what you mean by "the behavior" or by "slide to the right". I'm sorry.

**Reply by Subnet-Masked** [reply, marked as answer]
Sorry let me try and demonstrate, I did find a CSS entry that let me change it...
First image is currently how it works, the title of the current page moves over to the right in the nav and no longer lines up with the rest of the Nav. Second image is how it looks in 9.4.1, the title of the current page stays in place.

The css I changed is:
```
 .md-nav__item--active{
    padding: 0 !important;
 }
```
![image](https://github.com/squidfunk/mkdocs-material/assets/55600371/2f569dc8-fafc-455b-ad0f-2fb6bfa9d806)

![image](https://github.com/squidfunk/mkdocs-material/assets/55600371/3acfc2c4-8865-46c2-b913-e94a803bb61a)

**Reply by Subnet-Masked** [reply]
I guess now I found a solution I am curious if there is a more official way or if I should stick with the CSS edit :)
